src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditorContainer.js

```diff
@@ -46,6 +47,11 @@ class OrderTemplatesEditorContainer extends Component {
     addresses: ADDRESSES,
     vendors: VENDORS,
     materialTypes: MATERIAL_TYPES,
+    orderTemplate: {
```

Comment:
please extends https://github.com/folio-org/ui-orders/pull/487/files#diff-582afa5c4ad946c3823c99898b4b666eR41 with `fetch: false`

Reply:
shouldn't we use some `baseResource` everywhere?

Reply:
@NikitaSedyx I don't see a purpose of `fetch: false` here. It works fine on GET by id. In case when param `id` is absent (create)  - resource will be not called on fetch

Reply:
@aliaksei-chumakou `orderTemplates` resource loads list, for this `fetch: false` should be used if 2 resources are used

Reply:
@NikitaSedyx this resource is used to load template - `get(orderTemplate, 0, {})` below

Reply:
I'm not about this, there are 2 resources now

Reply:
@AndreiShumski2  https://github.com/folio-org/ui-invoice/blob/master/src/common/resources/invoice.js#L12 
it looks like you need something like `invoiceLineResource`

Reply:
fixed